src/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -1366,6 +1372,11 @@ internal byte[] GetData(string item)
             throw new Win32Exception(error);
         }
 
+        internal void ReleaseData(byte[] data)
+        {
+            perfDataKey.ReleaseBlob(data);
```

Comment:
`blob` and `data` seem used for essentially the same stuff. Would it be better to use `data` consistently? At least this place should be `ReleaseData` because it's being passed `data`